docs/perf_analyzer.md

```diff
@@ -400,3 +400,39 @@ can achieve by using shared memory in your application. Use
 By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
 protocol can be specificed with the -i option. If GRPC is selected the
 --streaming option can also be specified for GRPC streaming.
+
+## Using C API directly
+
+### Prerequisite
+Have the Triton SDK container and the Inference Server container on your machine
+Since you will need access to the Tritonserver install, it might be easier if 
+you copy the perf_analyzer binary to the Inference Server container.
+
+### Required Params
+Use the --help option to see complete documentation for how to use the C API.
+By default perf_analyzer relies on the triton instance running. You can select
+the C API mode with the --service-kind option. In additon, you will need to point
+perf_analyzer to the triton library path (e.g. /opt/tritonserver) and the model 
+repository path (e.g. /data/).
+A example run would look like:
+```
+perf_analyzer -m graphdef_int32_int32_int32 \
+--service-kind=triton_c_api \
+--model-repo=/workspace/qa/L0_perf_analyzer_capi/models \
+--library-name=/opt/tritonserver
```

Comment:
This argument should not be called --library-name, it should be something like --triton-path or --triton-directory